contracts/MarketTopShotOldVersion.cdc

```diff
@@ -37,37 +37,41 @@ import DapperUtilityCoin from 0xDUCADDRESS
 import NonFungibleToken from 0xNFTADDRESS
 import TopShot from 0xTOPSHOTADDRESS
 
-pub contract Market {
+access(all) contract Market {
+
+    access(all) entitlement Create
+    access(all) entitlement Withdraw
```

Comment:
I would probably recommend removing `Withdraw` here and just using `NonFungibleToken.Withdraw` instead